src/metadata.yaml

```diff
@@ -11,7 +11,7 @@ tags:
   - ldap
 series:
   - bionic
-  - eoan
+  - focal
```

Comment:
FTR, I'm fine with pre-emptively enabling the dev series metadata as always.

Note that it is not tested under this current proposal, and we will need to enable that soon as a matter of general ongoing dev/maintenance.